### ISSUE TITLE
ISO Cache -> Black screen in God Eater Burst JP v1.01

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#285]
```
What steps will reproduce the problem?
1. Patch a copy of God Eater Burst JP ISO with the v1.01 pboot.bin using 
isotool.
2. Start the game with ISO Cache enabled.

What is the expected output? What do you see instead?
Game should boot up normally, but does not unless ISO Cache is disabled in the 
recovery menu. A black screen occurs.

What version of the product are you using? On what operating system?
6.20PROB9, PSP3000, NoDRM engine enabled, Inferno and SonyNP9660 driver tried.

Please provide any additional information below.
Maybe ISO Cache doesn't work with ISOs that have been updated with pboots. Such 
as LittleBigPlanet and Phantasy Star Portable 2 JP. Or could be another cause. 
Fix appreciated.
```

Original issue reported on code.google.com by `morrn...@gmail.com` on 12 Sep 2011 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
Try the nightly and see if iso catching works ok their was a fix for another 
game for catching and it might help that as well

http://code.google.com/p/procfw/wiki/NightlyBuilds 
```

Original comment by `randqalan@gmail.com` on 12 Sep 2011 at 11:22

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the reply.
Didn't work, still resulted in a blackscreen. 
6.20 PRO Nightly (at time of writing)
Inferno/Sony NP9660: Still Black screen if ISO Cache enabled
m33driver: No problem as no ISO Cache (easy way to quicktest problem)
Sidenote: New flashing bar for recovery menu woo.
```

Original comment by `morrn...@gmail.com` on 13 Sep 2011 at 4:55

**Comment by GoogleCodeExporter** [comment]
```
Have you tried changing ISO cache size read
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
No result.

Changing each setting one by one:
Use ISO Cache: Enabled/Disabled (Default/Turn off) <-Disable work
Cache Size: 1, 5, 10, 15, 20, 23 (Default 20ish/5 intervals) <-Still no work
Cache Number: 64, 128, 640 (Default 128/extremes) <-Still no work
Cache Policy: LRU/Random No fix (Default/Other) <-Still no work

```

Original comment by `morrn...@gmail.com` on 13 Sep 2011 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
I guess the patch did mess up the iso Cache
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
Just wonder thou did you decrypt eboot.bin before you patched it maybe the 
patch added new encryption just a suggestion   
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 8:38

**Comment by GoogleCodeExporter** [comment]
```
Try build 1198 (bc3b764e3aa9) ISOCache: fixed infinite loop if reached EOF of 
ISO.
```

Original comment by `outma...@gmail.com` on 12 Oct 2011 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the heads-up, I'll try it out tomorrow 
when the server updates the Nightly Build with it.
```

Original comment by `morrn...@gmail.com` on 12 Oct 2011 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
Issue has been resolved, able to startup God Eater Burst JP v1.01 ISO 
with ISO Cache enabled after updating to NightlyBuild for 6.20.
Both Inferno and NP9660 driver working.
Cleared a mission with no problems.
Thanks hrimfaxi!
```

Original comment by `morrn...@gmail.com` on 13 Oct 2011 at 9:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `outma...@gmail.com` on 17 Oct 2011 at 9:43
- Changed state: **Fixed**
